opt/prepare-aiidalab.sh

```diff
@@ -6,6 +6,18 @@ set -x
 # Environment.
 export SHELL=/bin/bash
 
+# Activating conda
```

Comment:
is this not taken care of by the aiida-core container?

Reply:
No, this is not how it works.

Conda puts some activation code in the `.bashrc` which is not always loaded. For example, if I run something as another user from root, e.g. `su -c 'some command' aiida, the `.bashrc` file is not loaded. This is the case here.

Reply:
do we have to replicate this or can we just source the `.bashrc` then ?